### PR TITLE
Override social sharing image

### PR DESCRIPTION
**Pull request by hillaryj**
Adds a category-specific social sharing image (i.e. mortgages).

### CODE REVIEW

File: src/_layouts/base.html [hunk 1]
```diff
@@ -39,9 +39,9 @@
 <meta property="og:type" content="{% block og_type %}website{% endblock %}">
 <meta property="og:url" content="{{ request.url }}">
 <meta property="og:image"
-      content="{{ request.scheme }}://{{ request.get_host() }}{{ static('img/logo_open-graph_facebook.png') }}">
+      content="http://files.consumerfinance.gov/f/images/ask-cfpb-social-mortgages.original.png">
```
Comment (sebworks) [inline src/_layouts/base.html]: should these be https?

Comment (hillaryj) [reply src/_layouts/base.html]: Unfortunately, no - https can't resolve them.

Comment (sebworks) [reply src/_layouts/base.html]: try: https://s3.amazonaws.com/files.consumerfinance.gov/f/images/ask-cfpb-social-mortgages.original.png

Comment (hillaryj) [reply src/_layouts/base.html]: Will do!

**Comment by sebworks** [comment]
Looks good, merge at will.

**Pull request merged**